lua/neo-tree/defaults.lua

```diff
@@ -38,6 +38,43 @@ local config = {
   sort_function = nil , -- uses a custom function for sorting files and directories in the tree
   use_popups_for_input = true, -- If false, inputs will use vim.ui.input() instead of custom floats.
   use_default_mappings = true,
+  source_selector = {
+    winbar = true, -- toggle to show selector on winbar
+    statusline = false, -- toggle to show selector on statusline
+    tab_labels = { -- falls back to source_name if nil
+      filesystem = " Files",
+      buffers = " Buffers",
+      git_status = " Git",
+    },
+    content_layout = "center", -- only with `tabs_layout` = "equal", "focus"
+    --                start  : |/ 裡 bufname     \/...
+    --                end    : |/     裡 bufname \/...
+    --                center : |/   裡 bufname   \/...
+    tabs_layout = "equal", -- start, end, center, equal, focus
+    --             start  : |/  a  \/  b  \/  c  \            |
+    --             end    : |            /  a  \/  b  \/  c  \|
+    --             center : |      /  a  \/  b  \/  c  \      |
+    --             equal  : |/    a    \/    b    \/    c    \|
+    --             active : |/  focused tab    \/  b  \/  c  \|
+    text_trunc_to_fit = true, -- toggle to truncate text in each tab when not enough width
+    padding = 0, -- can be int or table
+    -- padding = { left = 2, right = 0 },
+    separator = "|", -- can be string or table, see bellow
```

Comment:
```suggestion
    separator = "│", -- can be string or table, see below
```
I think this is a nicer character to use here as it actually spans the full cell height, but up to you

Reply:
Thanks! Actually I tried to find that char but couldn't find it